fjord/alerts/api_views.py

```diff
@@ -189,26 +189,29 @@ def post(self, request):
         try:
             flavorslug = request.DATA['flavor']
         except KeyError:
-            return self.rest_error(
-                status=404,
-                errors='Flavor not specified in payload.'
-            )
+            raise exceptions.ParseError({
+                'flavor': [
+                    'Flavor not specified in payload'
+                ]
+            })
 
         try:
             flavor = AlertFlavor.objects.get(slug=flavorslug)
```

Comment:
This is the POST API which takes a single flavor and doesn't currently use a serializer.